types/three/three-core.d.ts

```diff
@@ -5273,13 +5273,23 @@ export class Line extends Object3D {
     geometry: Geometry | BufferGeometry;
     material: Material | Material[];
 
-    type: "Line";
+    type: "Line" | "LineLoop";
```

Comment:
This [can become `"LineSegments"` too](https://github.com/mrdoob/three.js/blob/master/src/objects/LineSegments.js#L13), but I guess nobody noticed it? 🤔